mqt/benchviewer/src/backend.py

```diff
@@ -150,14 +150,66 @@ def createDatabase(zip_file: ZipFile):
     return database
 
 
-def read_mqtbench_all_zip():
+def read_mqtbench_all_zip(
+    skip_question: bool = False,
+    target_location: str = "mqt/benchviewer/static/files/qasm_output",
+):
     global MQTBENCH_ALL_ZIP
-    huge_zip = Path("./static/files/qasm_output/MQTBench_all.zip")
+    huge_zip = Path(target_location + "/MQTBench_all.zip")
+    version_of_suitable_benchmark_zipfile = "v1.0.0"
+
+    url = (
+        "https://api.github.com/repos/nquetschlich/test/releases/tags/"
```

Comment:
do not forget to adapt this before the PR is being merged.

technically, you could also add the current zip-file to the existing release in MQTBench and test it that way (on the "live subject")

Reply:
done